server/form-pages/apply/risk-management-features/rehabilitativeInterventions.ts

```diff
@@ -0,0 +1,76 @@
+import type { TaskListErrors } from '@approved-premises/ui'
+import { Application } from '../../../@types/shared'
+import { convertKeyValuePairToCheckBoxItems } from '../../../utils/formUtils'
+
+import TasklistPage from '../../tasklistPage'
+
+export const interventions = {
+  accomodation: 'Accomodation',
+  drugsAndAlcohol: 'Drugs and alcohol',
+  childrenAndFamilies: 'Children and families',
+  health: 'Health',
+  educationTrainingAndEmployment: 'Education, training and employment',
+  financeBenefitsAndDebt: 'Finance, benefits and debt',
+  attitudesAndBehaviour: 'Attitudes, thinking and behaviour',
+  abuse: 'Abuse',
+  other: 'Other',
+} as const
+
+type Interventions = Array<keyof typeof interventions> | keyof typeof interventions
+
+export default class RehabilitativeInterventions implements TasklistPage {
+  name = 'rehabilitative-interventions'
+
+  title = `Which rehabilitative interventions will support the Approved Premises (AP) placement?`
+
+  body: { interventions: Interventions; otherIntervention?: string }
+
+  constructor(
+    body: Record<string, unknown>,
+    private readonly _application: Application,
+    private readonly previousPage: string,
+  ) {
+    if (this.otherInterventionIsNeeded(body)) {
+      this.body = {
+        interventions: body.interventions as Interventions,
+        otherIntervention: body.otherIntervention as string,
+      }
+    } else {
+      this.body = { interventions: body.interventions as Interventions }
+    }
+  }
+
+  previous() {
+    return this.previousPage
+  }
+
+  next() {
+    return ''
+  }
+
+  response() {
+    return {
+      [this.title]: {
+        Interventions: Array.isArray(this.body.interventions)
```

Comment:
I'm struggling to see here where `this.body.interventions` will ever not be an array? If there's a condition where it will be, would it perhaps be better if we put `body.interventions` into and array and flattened it, to ensure we always have an array, instead of having a complex conditional?

Reply:
If the user ticks one checkbox the response will be just that string. But good shout: we should put that into an array to save messiness elsewhere